src/main/java/Parser.java

```diff
@@ -0,0 +1,42 @@
+public class Parser {
+    public static Command parse(String input) throws ToDoException, DeadlineException, EventException, 
```

Comment:
This chunk of code has quite a lot of logic in it 🧠
While that's totally fine, I would suggest to refactor part of the logic, namely (parsing index of task and getting task description) into their own separate methods.

Doing so will have a few benefits:
1. Abstracting into methods gives you opportunity to document them. Naming them would also make it more readable for the reader
2. Methods are unit testable. This gives more opportunity for unit testing and makes your code less fragile and look more reliable 💪